src/Api/Customers.php

```diff
@@ -71,6 +71,11 @@ public function listPackages($customerIdOrUrlName)
         return $this->get(sprintf('/customers/%s/packages/', $customerIdOrUrlName));
     }
 
+    public function listCustomerPackage($customerIdOrUrlName, $packageName)
```

Comment:
Do you think this should be renamed to `showCustomerPackage()` instead? Would be consistent with `$client->customers()->show()` and `$client->packages()->showPackage()` :thinking:

Reply:
Yes, adjusting the name would be great. Looking at the other methods it should be `$client->customers()->showPackage()`